### ISSUE TITLE
Bug in analogRead

### ISSUE THREAD

**Issue by sbstjn** [ecto/duino#7]
Hi, there is a little bug in your `board.analogRead`, the code should be:

```
Board.prototype.analogRead = function (pin) {
    pin = this.normalizePin(pin);
    this.log('info', 'analogRead from pin ' + pin);
    this.write('04' + pin + this.normalizeVal(0));
}
```

I'm sorry I haven't created a pull request, but my repository is too messed up right now :D

**Comment by ecto** [comment]
Good to hear you're hacking on duino! I'll make a fix when I get a few spare minutes

**Comment by sbstjn** [comment]
I've started building a framework for piping communication with duino through a WebSocket so you can build HTML interfaces and use a browser to control the Arduino. You should have a look at my [Noduino](http://semu.github.com/noduino/) project, duino is an import core component of it ;)

**Comment by ecto** [comment]
fixed in merge #2
